drivers/spi/spi_ll_stm32.h

```diff
@@ -1,26 +0,0 @@
-/*
```

Comment:
Is it strictly necessary to remove this header for this change ?

Reply:
No it is not necessary. It just happened over the various adaptations to the changing requirements to codegen.